docs/otel/export-metrics.md

```diff
@@ -0,0 +1,64 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: OTel Export
+--->
+
+# Semantic Conventions for OTel Export Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+This document describes instruments and attributes for OTel
+Export level metrics. Consider the [general metric semantic
+conventions](README.md#general-metric-semantic-conventions) when creating
+instruments not explicitly defined in the specification.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Metric Instruments](#metric-instruments)
+  * [Metric: `otel.processor.spans`](#metric-otelprocessorspans)
+  * [Metric: `otel.exporter.spans`](#metric-otelexporterspans)
+
+<!-- tocstop -->
+
+## Metric Instruments
+
+### Metric: `otel.processor.spans`
+
+This metric is [required][MetricRequired].
+
+<!-- semconv metric.otel.processor.spans(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `otel.processor.spans` | Counter | `{span}` | Measures the number of processed Spans. |
+<!-- endsemconv -->
+
+<!-- semconv metric.otel.processor.spans(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `processor.dropped` | boolean | Whether the Span was dropped or not. [1] |  | Required |
+| `processor.type` | string | Type of processor being used. | `BatchSpanProcessor` | Recommended |
+
+**[1]:** Spans may be dropped if the internal buffer is full.
+<!-- endsemconv -->
+
+### Metric: `otel.exporter.spans`
+
+This metric is [required][MetricRequired].
+
+<!-- semconv metric.otel.exporter.spans(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `otel.exporter.spans` | Counter | `{span}` | Measures the number of exported Spans. |
+<!-- endsemconv -->
+
+<!-- semconv metric.otel.exporter.spans(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `exporter.dropped` | boolean | Whether the Span was dropped or not. [1] |  | Required |
```

Comment:
There are 3 states from the exporter's view:
1. delivered (the exporter has received the confirmation from the ingestion)
2. dropped (the exporter decided to drop the data knowing that the ingestion didn't accept it)
3. unknown (the exporter has sent the data, but it has no idea whether the data is accepted or not by the ingestion)

Reply:
Oh, good one, yes. So I guess my initial approached took for granted this:

- `dropped=false`: Delivered, with confirmation from the ingestion step.
- `dropped=true`: Either consecutive attempts were exhausted OR it's not confirmed ingestion was successful (including error on the server side).

Think it would make sense to have 3 values instead of two here (as suggested by your comment)?

Reply:
(On a related note: I will follow up this PR, once it's merged, but some additional metrics, such as retries count at the export level, etc)